src/view/channel-mobile.js

```diff
@@ -92,21 +92,13 @@ ChannelList.propTypes = {
 
 const iStyles = StyleSheet.create({
   item: {
-    flexDirection: 'column',
     alignItems: 'stretch',
-    height: 180,
-    paddingRight: 10,
-    paddingLeft: 10,
-    marginBottom: 15,
-    backgroundColor: color.glasDarker,
-    borderBottomWidth: 0,
-    borderRadius: 7,
```

Comment:
I just saw that the new mobile list items all had fixed height. The new CardItem is a bit more flexible.